xbmc/addons/AddonManager.h

```diff
@@ -124,14 +124,24 @@ namespace ADDON
      */
     bool FindAddons();
 
+    /*!
+     * @brief Fills the the provided vector with the list of incompatible
+     * enabled addons and returns if there's any.
+     *
+     * @param[out] incompatible List of incompatible addons
+     * @return true if there are incompatible addons
+     */
+    bool GetIncompatibleEnabledAddons(std::vector<AddonInfoPtr>& incompatible) const;
+
     /*!
      * @brief Fills the the provided vector with the list of incompatible
      * addons and returns if there's any.
      *
      * @param[out] incompatible List of incompatible addons
+     * @param[in] whether or not to include incompatible addons that are disabled
      * @return true if there are incompatible addons
      */
-    bool GetIncompatibleAddons(std::vector<AddonInfoPtr>& incompatible) const;
+    bool GetIncompatibleAddons(std::vector<AddonInfoPtr>& incompatible, bool includeDisabled) const;
```

Comment:
Maybe better to rename this and the new then to `...AddonInfos` on end, and to use the `...Addons` only for them wher give the **`CAddon/AddonPtr`**.

So from function name can direct seen which part it give.

Reply:
Done